### PR TITLE
[AN-4414] Add like to message long press menu

### PR DESCRIPTION
**Pull request by hakonbo**
#### APK
[Download build #7435](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7435/artifact/build/artifact/wire-dev-PR85-7435.apk)
[Download build #7436](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7436/artifact/build/artifact/wire-dev-PR85-7436.apk)
[Download build #7437](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7437/artifact/build/artifact/wire-dev-PR85-7437.apk)
[Download build #7439](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7439/artifact/build/artifact/wire-dev-PR85-7439.apk)
[Download build #7440](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7440/artifact/build/artifact/wire-dev-PR85-7440.apk)

### CODE REVIEW

**Comment by hakonbo** [comment]
build again

**Comment by hakonbo** [comment]
build again

File: app/src/main/java/com/waz/zclient/pages/main/conversation/views/MessageBottomSheetDialog.java [hunk 1]
```diff
@@ -100,6 +109,23 @@ public void onClick(View v) {
         root.addView(row, params);
     }
 
+    private boolean isLikeAllowed() {
+        switch (message.getMessageType()) {
```
Comment (shumengye) [inline app/src/main/java/com/waz/zclient/pages/main/conversation/views/MessageBottomSheetDialog.java]: Also check if user is still member of conversation?

Comment (hakonbo) [inline app/src/main/java/com/waz/zclient/pages/main/conversation/views/MessageBottomSheetDialog.java]: Right, forgot about that. Will add.

**Pull request merged**